base/db/R/convert_input.R

```diff
@@ -77,7 +76,6 @@ convert_input <-
            fcn,
            con = con,
            host,
-           browndog,
```

Comment:
👍 to removing the argument, but this means this PR will also need to find and remove the `browndog` arg from all existing calls to `convert_input` elsewhere in PEcAn.

Reply:
Yes. That's what I'm willing to do by end of this week too.